src/shared/components/QuaiPayContactBottomSheet/QuaiPayContactBottomSheet.tsx

```diff
@@ -245,7 +242,7 @@ export const QuaiPayContactBottomSheet: React.FC = () => {
               onSearchChange={setSearchText}
               placeholder={t('home.send.searchByAddress')}
               onPress={expandBottomSheet}
-              onPressPaste={() => navigateToAmount(detectedAddress)}
+              onPressRightIcon={() => navigateToAmount()}
```

Comment:
you can avoid this arrow fx here and simply pass `navigateToAmount` (also it will avoid the anonymous fx 😉)

```suggestion
              onPressRightIcon={navigateToAmount}
```